src/exporter/models/geoPackage.ts

```diff
@@ -0,0 +1,13 @@
+import { types, Instance } from 'mobx-state-tree';
+
+export const geoPackage = types.model({
```

Comment:
rename data and file to exportRequestStatus or exportTaskStatus

Reply:
Renaming will be done in unit test PR, currently too much merging involved, sorry